### DISCUSSION TITLE
[Just asking question] How does Kavita storage works?

### DISCUSSION THREAD

**Discussion by spawnpeekboi** [Kareadita/Kavita#3170] [Ideas]
### Idea Description

As listed here https://wiki.kavitareader.com/troubleshooting/faq#q-i-really-like-kavita-but-can-it-mirror-or-just-use-my-folder-structure,
Kavita is not designed to use folder structure.
I just wanna to ask it works by copying every file in the library path and put it inside the docker image after reorganize it with its' own method? Or it works like immich external library which stores path to file in its database and reference to the actual location when asked for the file?
thx thx many thx

### Idea Category

User Experience

### Duration of Using Kavita

1 month

### Before submitting

- [ ] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
What that means is Kavita doesn't group or provide a folder/file view for you. 

You mount your library and Kavita will perform grouping based on parsed/read metadata. It does not duplicate your files or anything like that. It's similar to how Plex functions.
